src/App.jsx

```diff
@@ -19,9 +19,9 @@ export default function App() {
             <Route path="/login">
               <Login />
             </Route>
-            <Route path="/treehouse/members">
+            <PrivateRoute path="/treehouse/members">
```

Comment:
Also here updating your routes to Private with your context provider which makes state accessible to the entire app.